study-builder/studies/testboston/subs.conf

```diff
@@ -0,0 +1,32 @@
+{
+  "contact": {
+    "email": "testboston@datadonationplatform.org",
+    "phone": "XXX-XXX-XXXX"
+  },
+  "social": {
+    "twitter": "foo",
+    "facebook": "bar",
+    "instagram": "baz"
+  },
+  "i18n": {
+    "en": {
+      include required("i18n/en.conf")
+    }
+  },
+  "id": {
```

Comment:
Looking for the study guid? Or the activity code or question identifiers? It's now all in one place :)